aws/resource_aws_lambda_permission_test.go

```diff
@@ -188,6 +189,12 @@ func TestAccAWSLambdaPermission_basic(t *testing.T) {
 					resource.TestCheckResourceAttr("aws_lambda_permission.allow_cloudwatch", "event_source_token", "test-event-source-token"),
 				),
 			},
+			{
+				ResourceName:      "aws_lambda_permission.allow_cloudwatch",
+				ImportState:       true,
+				ImportStateId:     fmt.Sprintf("%s/%s", funcName, statementId),
+				ImportStateVerify: true,
+			},
```

Comment:
Hi @chaspy! Could you please add blocks like this to the rest of the acceptance tests in here, and re-run the acceptance tests once you've done that? Thanks!